analyses/compare-gistic/util/GISTIC-comparison-functions.R

```diff
@@ -191,33 +195,32 @@ plot_genes_venn_diagram_wrapper <- function(cohort_genes_file,
   #    medulloblastoma_genes_file: file path to the `amp_genes.conf_90.txt` or
   #                                `del_genes.conf_90.txt` file for the
   #                                 medulloblastoma histology
-  
-  # Run `get_genes_object` function on each of the files
-  cohort_genes_vector <- get_genes_object(cohort_genes_file)
-  lgat_genes_vector <- get_genes_object(lgat_genes_file)
-  hgat_genes_vector <- get_genes_object(hgat_genes_file)
-  medulloblastoma_genes_vector <- get_genes_object(medulloblastoma_genes_file)
-  
+
+  # Run `format_gistic_genes` function on each of the files
+  cohort_genes_vector <- format_gistic_genes(cohort_genes_file)
+  lgat_genes_vector <- format_gistic_genes(lgat_genes_file)
+  hgat_genes_vector <- format_gistic_genes(hgat_genes_file)
+  medulloblastoma_genes_vector <- format_gistic_genes(medulloblastoma_genes_file)
+
   # Run `plot_venn_diagram` for each comparison case
   lgat_venn <- plot_venn_diagram(cohort_genes_vector, lgat_genes_vector, "lgat_genes")
   hgat_venn <- plot_venn_diagram(cohort_genes_vector, hgat_genes_vector, "hgat_genes")
   medulloblastoma_venn <- plot_venn_diagram(cohort_genes_vector, medulloblastoma_genes_vector, "medulloblastoma_genes")
-  
+
   # Save plots to list
   venn_plot_list <- list(lgat_venn, hgat_venn, medulloblastoma_venn)
-  
+
   # Return the plot list
   return(venn_plot_list)
-  
 }
 
 #### Implemented in `02-GISTIC-tidy-data-prep.Rmd` ----------------------------
 
 prepare_gene_level_gistic <- function(all_lesions_file,
                                       amp_genes_file,
                                       del_genes_file,
-                                      output_filename) {
-  
+                                      gene_level_output_tsv_filename) {
```

Comment:
OH. This isn't what I meant. Sorry. Your argument name itself is fine. I was talking about when you call the function can you put `output_filename =` in front of it so I know that one is an output file and the others are input files.  Can you change it back to `output_filename`? Sorry if I didn't communicate my idea clearly.

Reply:
Ohh, okay gotcha! I should have double checked my understanding before making that change. I'll change the argument back and make that initial change that you suggested.